docs/Changelog.md

```diff
@@ -13965,6 +13965,134 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain input and output types to all numeric tensors.</dd>
 </dl>
 
+### <a name="BatchNormalization-12"></a>**BatchNormalization-12**</a>
+
+  Carries out batch normalization as described in the paper https://arxiv.org/abs/1502.03167.
+  There is three required inputs 'X', 'mean' and 'var', in addition to one optional input 'training_mode'.
+  Note that 'mean' and 'var' are expected to be the estimated statistics in inference mode (training_mode=False, default),
+  and the running statistics in training mode (traning_mode=True).
+  There is one required output 'Y' and four optional outputs : 'mean', 'var', 'saved_mean', 'saved_var' used for training.
+  
+  The statistics are updated as follows:
```

Comment:
```suggestion
  The statistics are updated as follows:
```
Would the update happen in inference mode? If the computation rules depend on `training_mode`, we need to have the equations listed for both cases (`training_mode`=True/False).